### DISCUSSION TITLE
Updated Sonoff TH10/TH16 PCB design

### DISCUSSION THREAD

**Discussion by apastuszak** [arendst/Tasmota#11027] [Q&A] [answered]
I just got a TH10 in the mail from Amazon, and the PCB is a little different.  Here is what it looks like:

![IMG_2465](https://user-images.githubusercontent.com/3401054/108381484-d9bdcf00-71d5-11eb-9ecd-61504785895e.jpeg)

**Comment by ascillato2** [comment]
Have you tested Tasmota on it? All gpios are the same as the actual template?

**Reply by apastuszak** [reply]
Yep. I flashed it last night.  Everything is the same.

**Comment by apastuszak** [comment, marked as answer]
I wonder what the exp-log pin is for.

**Reply by barbudor** [reply]
EXP-LOG Is probably a GPIO. Could you track it to the ESP to update the information page ?

**Reply by apastuszak** [reply]
I found a comment on a blog post that says it's GPIO2.

**Reply by sfromis** [reply]
https://tasmota.github.io/docs/devices/Sonoff-TH/

The docs already mention PCB Ver 2.1 (from 2019), and state that EXP-LOG is indeed GPIO2.
